apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.RetryConformanceTests/RetryConformanceTest.cs

```diff
@@ -370,6 +394,10 @@ void MaybeDeleteBucket(string bucketName)
             {
                 return;
             }
+            if (Client.GetBucket(bucketName) is null)
```

Comment:
I don't think GetBucket will ever return null. It will throw an exception if the bucket doesn't exist.